src/settings.js

```diff
@@ -72,6 +72,7 @@ defaults = {
   parallelDirectUploads: 10,
   passWindowOpen: false,
   // camera recording
+  preferedMimeType: null,
```

Comment:
This should be a list of ordered preferred options.
```suggestion
  videoPreferredMimeTypes: null,
```